### PR TITLE
Add new rules for ANSSI BP28 R22

### PR DESCRIPTION
**Pull request by jan-cerny**
These rules check network related sysctl settings. The R22 requires multiple settings, for some of them we already have rules in our project but for others we have to created new rules.

- sysctl_net_ipv6_conf_all_accept_ra_defrtr
- sysctl_net_ipv6_conf_all_accept_ra_pinfo
- sysctl_net_ipv6_conf_all_accept_ra_rtr_pref
- sysctl_net_ipv6_conf_all_autoconf
- sysctl_net_ipv6_conf_all_max_addresses
- sysctl_net_ipv6_conf_all_router_solicitations
- sysctl_net_ipv6_conf_default_accept_ra_defrtr
- sysctl_net_ipv6_conf_default_accept_ra_pinfo
- sysctl_net_ipv6_conf_default_accept_ra_rtr_pref
- sysctl_net_ipv6_conf_default_autoconf
- sysctl_net_ipv6_conf_default_max_addresses
- sysctl_net_ipv6_conf_default_router_solicitations
- sysctl_net_ipv4_ip_local_port_range
- sysctl_net_ipv4_tcp_rfc1337

The PR also extends the sysctl template with new parameters which was needed for rule sysctl_net_ipv4_ip_local_port_range so for this rule it adds also test scenarios.

### CODE REVIEW

**Comment by jan-cerny** [comment]
@openscap-ci test this please

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_router_solicitations_value.var [hunk 1]
```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: net.ipv6.conf.default.router_solicatations
+
+description: 'Accept all router solicatations by default?'
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_router_solicitations_value.var:5]: typo: `solicatations`.

File: linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_router_solicitations_value.var [hunk 2]
```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: net.ipv6.conf.default.router_solicatations
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_router_solicitations_value.var:3]: typo: `solicatations`

File: linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_ip_local_port_range/tests/comment.fail.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if grep -q "^net.ipv4.ip_local_port_range" /etc/sysctl.conf; then
+	sed -i "s/^net.ipv4.ip_local_port_range.*/# net.ipv4.ip_local_port_range = 32768 65535/" /etc/sysctl.conf
+else
+	echo "# net.ipv4.ip_local_port_range = 32768 65535" >> /etc/sysctl.conf
+fi
+# setting correct runtime value to check it properly
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_ip_local_port_range/tests/comment.fail.sh:8]: This comment is a bit misleading.
The test comments the file config, but sets correct runtime values.
Suggestion: 
` # setting correct runtime value to check if the filesystem configuration is evaluated properly`

**Review by yuumasato — COMMENTED** [review]
LGTM, just a few typos.

**Review by yuumasato — APPROVED** [review]

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
Issues with `ci/prow/e2e-aws-ocp4-cis` seem unrelated to this PR.

**Pull request merged**